store/store.go

```diff
@@ -735,6 +735,31 @@ func (s *Store) doRequest(ctx context.Context, client *http.Client, reqOptions *
 	}
 }
 
+func (s *Store) buildLocationString() (string, error) {
+	if s.dauthCtx == nil {
+		return "", nil
+	}
+
+	cloudInfo, err := s.dauthCtx.CloudInfo()
+	if err != nil {
+		return "", err
+	}
+
+	if cloudInfo != nil {
```

Comment:
Nitpick: I know you just moved the code, but it seems it would make sense to reverse the if check here:
```
if cloudInfo == nil {
    return "", nil 
}
...
```

Reply:
Then maybe even
```go
if err != nil || cloudInfo == nil {
    return "", err
}
```